actionview/lib/action_view/helpers/form_helper.rb

```diff
@@ -754,7 +754,11 @@ def form_with(model: nil, scope: nil, url: nil, format: nil, **options, &block)
 
         if model
           if url != false
-            url ||= polymorphic_path(model, format: format)
```

Comment:
I was curious to see if we could implement this as `polymorphic_path(model, { format: format }.compact)`.

But then I noticed that I can't get the test to fail if I change the code back to the version before this PR. @cryptoque can you double-check the test to see if it needs tweaking?

```
3.0.2 ~/code/rails/actionview main *= bin/test test/template/form_helper_test.rb -n /new_form_for_without_format/
Running 305 tests in parallel using 8 processes
Run options: -n /new_form_for_without_format/ --seed 41714

# Running:

.

Finished in 0.204844s, 4.8818 runs/s, 4.8818 assertions/s.
1 runs, 1 assertions, 0 failures, 0 errors, 0 skips
```

Reply:
@kaspth that is good point, I will modify the test on Monday.